willa/etl/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Defines the TIND API for Willa.
```

Comment:
This isn't the TIND API.  You probably want:

```suggestion
Provides ETL functionality for Willa.
```